integration/test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -77,6 +78,39 @@ func TestWriteLogsToCloudWatch(t *testing.T) {
 	}
 }
 
+// Validate https://github.com/aws/amazon-cloudwatch-agent/issues/447
```

Comment:
Might've mentioned this before, but I wonder if it is better to just put a quick description in case the link breaks.
I have definitely come across code with references to links that no longer exist and wished for something more.

1 more reason is that is saves developers reading the code from having to switch to a browser.

The downside is obviously that there is much more detail in the ticket and I don't expect it all to be captured in code comments.

Reply:
Somehow I missed this comment. That's fair. I will add descriptions where I tried to cheat by just linking the GH issue